.github/workflows/ci.yml

```
@@ -47,19 +51,23 @@ jobs:
 
       - name: ESLint
         run: |
-          changed_files=$(git diff --name-only --diff-filter=ACMRUXB origin/main)
+          changed_files=$(git diff --name-only --diff-filter=ACMRUXB origin/${{ github.base_ref }})
           
```

yamllint

[error] trailing spaces

(trailing-spaces)